cert_manager/metadata.csv

```diff
@@ -0,0 +1,9 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+cert_manager.prometheus.health,gauge,,,,Whether the check is able to connect to the metrics endpoint,0,cert_manager,cm.health
+cert_manager.certificate_ready_status,gauge,,,,The ready status of the certificate,0,cert_manager,cm.cert_ready_status
+cert_manager.certificate_expiration_timestamp,gauge,,second,,The date after which the certificate expires. Expressed as a Unix Epoch Time,0,cert_manager,cm.cert_exp_time
+cert_manager.acme_client_request_count,count,,,,The number of requests made by the ACME client,1,cert_manager,cm.acme_req_cnt
+cert_manager.acme_client_request_duration.sum,gauge,,,,The sum of the HTTP request latencies in seconds for the ACME client,1,cert_manager,cm.req_duration.sum
```

Comment:
```suggestion
cert_manager.acme_client.request_duration.sum,gauge,,,,The sum of the HTTP request latencies in seconds for the ACME client,1,cert_manager,cm.req_duration.sum
```

Reply:
@ChristineTChen Thanks for the suggestions. One question, though, why are you suggesting acme_client.request.count (3 levels), but acme_client.request_duration. I would like to be consistent throughout. Which one is recommended? Thanks!

Reply:
good catch! I'd go with `acme_client.request.duration`